library/src/com/google/maps/android/geoJsonLayer/GeoJsonFeature.java

```diff
@@ -0,0 +1,214 @@
+package com.google.maps.android.geoJsonLayer;
```

Comment:
no capitals in package name (http://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html). I might also just call it geojson instead of geojsonlayer